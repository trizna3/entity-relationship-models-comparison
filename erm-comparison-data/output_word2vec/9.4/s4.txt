vehicle-vehicles
line-lines
stop-stops
vehicle type-vehicle types
CHANGE_CARDINALITY:S-vehicle types-lines
MOVE_ATTR_TO_INCIDENT_ASSOCIATION:S-time-rides on
