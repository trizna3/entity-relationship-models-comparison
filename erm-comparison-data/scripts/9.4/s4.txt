entity set stops {
	"name"
};

entity set lines {
	"number"
};

entity set "first stops" {
	"numerical code"
};

entity set vehicles {
	"numerical code"
};

entity set "vehicle types" {
	"time"
};

association "line routes" between {
	stops MANY
	lines MANY
}{
	"stop order"
	"duration since first stop"
};

association "rides on" between {
	lines MANY
	"vehicle types" ONE
}{};

association "is of type" between {
	"vehicle types" ONE
	vehicles MANY
}{};

association "schedule" between {
	lines ONE
	"vehicle types" ONE
	vehicles MANY
	"first stops" ONE
}{
	"first stop leave time"
};