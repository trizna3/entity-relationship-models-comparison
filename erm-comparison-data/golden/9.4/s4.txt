vehicle-vehicles
vehicle type-vehicle types
line-lines
stop-stops